ranger/data/scope.sh

```diff
@@ -92,7 +92,15 @@ handle_extension() {
         json)
             jq --color-output . "${FILE_PATH}" && exit 5
             python -m json.tool -- "${FILE_PATH}" && exit 5
-            ;;
+            ;; # Continue with next handler on failure
+	## Jupyter Notebook
+        ipynb)
+            ## Convert to HTML then preview as text conversion
+	    jupyter nbconvert "${FILE_PATH}" --output=/tmp/notebook.html
```

Comment:
Let's use `mktemp` rather than hardcoding this path. Ideally we'd get a tmp directory to store the html or the text in every time so it's cached. (Not sure that's worth the trouble though, if this is fast enough let's not worry about it.)

Reply:
Good idea, I've used `mktemp -d` to create a temp dir, looks like it works well with caching.